src/main/java/org/broadinstitute/hellbender/tools/exome/FilterTargets.java

```diff
@@ -317,9 +330,159 @@ public String reasonToFilter(final Target target) {
     }
 
     /**
+     * Class that represent a interval of targets to exclude.
+     *
+     * It contains a copy of the interval specification (e.g. "chr1", "chr1:1+", "chr2:100,000,000-200,000,000").
+     *
+     * And the location where it was specified so that we can provide more informative error and target rejection messages.
+     */
+    private final class ExcludeInterval implements Locatable {
```

Comment:
Is done and it works. The code added complexity is minimal. Sure not vital but it does not harm IMO. 
If you insist I will remove; let me know if that is the case.